prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+#card-deck {
+  display: flex;
+  flex-basis: calc(100% / 3);
```

Comment:
You can target the elements with the class attribute of `"card-container"` with a `.` (dot).

```css
.card-container {

}
```

Try moving your `flex-basis` property to that rule-set.